### ISSUE TITLE
Dev branch build fail when libsdl < 2.0.18

### ISSUE THREAD

**Issue by npes87184** [Genymobile/scrcpy#2924]
```
[17/85] Compiling C object app/scrcpy.p/src_keyboard_inject.c.o                                                                 
FAILED: app/scrcpy.p/src_keyboard_inject.c.o                 
cc -Iapp/scrcpy.p -Iapp -I../app -I../app/src -I/usr/include/x86_64-linux-gnu -I/usr/local/include/SDL2 -I/usr/include/libusb-1.0 -f
lto -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c11 -g -D_REENTRANT -MD -MQ app/scrcpy.
p/src_keyboard_inject.c.o -MF app/scrcpy.p/src_keyboard_inject.c.o.d -o app/scrcpy.p/src_keyboard_inject.c.o -c ../app/src/keyboard_
inject.c                                                                                
In file included from ../app/src/trait/key_processor.h:9:0,
                 from ../app/src/keyboard_inject.h:10,
                 from ../app/src/keyboard_inject.c:1: 
../app/src/input_events.h:57:21: error: ‘KMOD_SCROLL’ undeclared here (not in a function); did you mean ‘KMOD_RALT’?
     SC_MOD_SCROLL = KMOD_SCROLL,                                                                                                   
                     ^~~~~~~~~~~                                                                                                    
                     KMOD_RALT        
```
KMOD_SCROLL is added in 2.0.18 (see https://github.com/libsdl-org/SDL/commit/cb1e20b058f1de55b98f32b7d16cff1f896f88dd)

**Comment by rom1v** [comment]
Thank you :+1:

Fixed by a9d23400cdb763703a6d757b0c445e7700271abb.

I quickly checked enum values from SDL 2.0.5, the remaining ones should be ok.

**Comment by npes87184** [comment]
It works!

Thank you.
